packages/react-components/react-migration-v0-v9/package.json

```diff
@@ -39,7 +39,8 @@
     "@griffel/react": "^1.5.2",
     "tslib": "^2.1.0",
     "@fluentui/react-components": "^9.15.6",
-    "@fluentui/react-northstar": "^0.66.2"
+    "@fluentui/react-northstar": "^0.66.2",
+    "lodash": "^4.17.15"
```

Comment:
```suggestion
```

We can use modern JS now, let's not add lodash 😬😬😬

Reply:
Removed